packages/flow-type-tests/component-base.js

```diff
@@ -0,0 +1,59 @@
+// @flow
```

Comment:
Is this file for manually spotchecking Flow?

Reply:
Yep, the flow is using the --max-warnings=0 flag so if there are unused suppressions this will cause an error as well (which can catch when something *should* error but does not).